hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -874,6 +874,22 @@
       run_timeout_mins: 120
       reseed: 1
     }
+    {
+      name: chip_sw_otp_ctrl_rot_auth_config
+      uvm_test_seq: chip_sw_rom_e2e_jtag_inject_vseq
+      sw_images: [
+        "//hw/ip/otp_ctrl/data/earlgrey_skus/emulation:otp_img_test_unlocked0_manuf_empty:4",
```

Comment:
This label will change to //hw/top_earlgrey/data/otp/sival_skus:otp_img_... once the #25503 is merged.